.github/workflows/daily-wpt-fyi.yml

```diff
@@ -0,0 +1,108 @@
+# This workflow runs every night and tests various releases of Node.js
+# (latest nightly, current, and two latest LTS release lines) against the
+# `epochs/daily` branch of WPT.
+
+name: Daily WPT report
+
+on:
+  workflow_dispatch:
+    inputs:
+      node-versions:
+        description: 'Node.js versions (as supported by actions/setup-node) to test as JSON array'
+        required: false
+        default: '["current", "lts/*", "lts/-1"]'
+  schedule:
+    # This is 20 minutes after `epochs/daily` branch is triggered to be created
+    # in WPT repo.
+    # https://github.com/web-platform-tests/wpt/blob/master/.github/workflows/epochs.yml
+    - cron: '30 0 * * *'
+
+env:
+  PYTHON_VERSION: '3.11'
+  WPT_FYI_ENDPOINT: ${{ vars.WPT_FYI_ENDPOINT }}
+  WPT_FYI_USERNAME: ${{ vars.WPT_FYI_USERNAME }}
+  WPT_FYI_PASSWORD: ${{ secrets.WPT_FYI_PASSWORD }}
```

Comment:
I'd set these only in the step that needs them (for safety, especially for the secret)